pkg/event/event.go

```diff
@@ -0,0 +1,204 @@
+package event
+
+import (
+	"github.com/openshift/linuxptp-daemon/pkg/daemon"
```

Comment:
This causes a import cycle. Should define the referred constants in event or a common package.

```
./hack/build.sh
package github.com/openshift/linuxptp-daemon/cmd
	imports github.com/openshift/linuxptp-daemon/pkg/daemon
	imports github.com/openshift/linuxptp-daemon/pkg/event
	imports github.com/openshift/linuxptp-daemon/pkg/daemon: import cycle not allowed
```

Reply:
fixed it